packages/reactive-element/src/decorators/query-assigned-elements.ts

```diff
@@ -50,9 +56,7 @@ export interface QueryAssignedElementsOptions
  * Note, the type of this property should be annotated as `Array<HTMLElement>`.
  *
  * @param options Object that sets options for nodes to be returned. See
- *     [MDN parameters section](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedElements#parameters)
- *     for available options. Also accepts two more optional properties,
- *     `slot` and `selector`.
+ *     {@link QueryAssignedElementsOptions} for all available options.
  * @param options.slot Name of the slot. Undefined or empty string for the
```

Comment:
Should we even keep this style of deep property docs when we should have docs on the QueryAssignedElementsOptions interface? Seems like this is duplication that will need to be synced.

Reply:
It's true. It also makes for a lot of info duplication. Removing the `options.slot`.

Reply:
Done!